x/stakeibc/keeper/hooks.go

```diff
@@ -58,27 +60,52 @@ func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochIdentifier string, epochN
 						continue
 					}
 					delegateAddress := delegateAccount.Address
+					// PROCESS TRANSFERS
 					// TODO(TEST-89): Set NewHeight relative to the most recent known gaia height (based on the LC)
 					// TODO(TEST-90): why do we have two gaia LCs?
-					timeoutHeight := clienttypes.NewHeight(0, 1000000)
-					transferCoin := sdk.NewCoin(hostZone.GetIBCDenom(), sdk.NewInt(int64(depositRecord.Amount)))
-					goCtx := sdk.WrapSDKContext(ctx)
-	
-					msg := ibctypes.NewMsgTransfer("transfer", hostZone.TransferChannelId, transferCoin, addr, delegateAddress, timeoutHeight, 0)
-					_, err := k.transferKeeper.Transfer(goCtx, msg)
-					if err != nil {
-						pstr := fmt.Sprintf("\tERROR WITH DEPOSIT RECEIPT {%d}", depositRecord.Id)
-						k.Logger(ctx).Info(pstr)
-						panic(err)
+					if depositRecord.Status == types.DepositRecord_TRANSFER {
```

Comment:
Why move the staking logic to hook rather than keep it in a separate function? Might be cleaner to separate it out, but I assume you have good reason to put it here

Reply:
Nvm, looks like you addressed this in the next commit 👍